### ISSUE TITLE
examples/cicd: GitHub Actions running 5 jobs in parallel (each tests every 5th file)

### ISSUE THREAD

**Issue by ulises-jeremias** [nanlabs/devops-reference#49]

**Comment by ulises-jeremias** [comment]
Closing as duplicate of #231, which keeps the GitHub Actions matrix sharding request in a clearer contributor-friendly format.
